wordpress-mobile/WordPress-Android
# Feature/update threaded comment design

Fixes #15593
Fixes #15594

This PR implements the first part of UI change to threaded comments. Comment moderation for Administrators will come later.

Share functionality for now only shares the shortened URL to the comment.

[[![Image from Gyazo](https://i.gyazo.com/90af003f1a2c1193863ac1e2e74e91f9.png)](https://gyazo.com/90af003f1a2c1193863ac1e2e74e91f9)](https://i.gyazo.com/65484547567cbf4f82450fd57f696773.png)

[![Image from Gyazo](https://i.gyazo.com/1ac4b881ca9609d3a5ec00a311832886.png)](https://gyazo.com/1ac4b881ca9609d3a5ec00a311832886)

To test:
- Open comments for a Post from the reader.
- Make sure they look similar to the screenshots provided.
- Confirm that the sharing button works as expected.

## Regression Notes
1. Potential unintended areas of impact
- Visual issue.
- Sharing now working.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual testing.

3. What automated tests I added (or what prevented me from doing so)
Changes are visual, so no testing is required.

PR submission checklist:
